maths/gcd_of_n_numbers.py

```diff
@@ -0,0 +1,110 @@
+"""
+Gcd of N Numbers
+Reference: https://en.wikipedia.org/wiki/Greatest_common_divisor
+"""
+
+from collections import Counter
+
+
+def get_factors(
+    number: int, factors: Counter | None = None, factor: int = 2
+) -> Counter:
+    """
+    this is a recursive function for get all factors of number
+    >>> get_factors(45)
+    Counter({3: 2, 5: 1})
+    >>> get_factors(2520)
+    Counter({2: 3, 3: 2, 5: 1, 7: 1})
+    >>> get_factors(23)
+    Counter({23: 1})
+    >>> get_factors(0)
+    Traceback (most recent call last):
+        ...
+    TypeError: number must be integer and greater than zero
+    >>> get_factors(-1)
+    Traceback (most recent call last):
+        ...
+    TypeError: number must be integer and greater than zero
+    >>> get_factors(1.5)
+    Traceback (most recent call last):
+        ...
+    TypeError: number must be integer and greater than zero
+
+    factor can be all numbers from 2 to number that we check if number % factor == 0
+    if it is equal to zero, we check again with number // factor
+    else we increase factor by one
+    """
+
+    match number:
```

Comment:
@dhruvmanila AlgorithemsKeeper is running on Py3.10, not 3.11.

Reply:
Must I change the code?

Reply:
Yes

Reply:
This should be fixed.

Reply:
@dhruvmanila can you help me that how can I fix it? It runs correctly on my computer